Extension/src/LanguageServer/configurations.ts

```diff
@@ -628,6 +628,27 @@ export class CppProperties {
             configuration.cStandard = this.updateConfigurationString(configuration.cStandard, settings.defaultCStandard, env);
             configuration.cppStandard = this.updateConfigurationString(configuration.cppStandard, settings.defaultCppStandard, env);
             configuration.intelliSenseMode = this.updateConfigurationString(configuration.intelliSenseMode, settings.defaultIntelliSenseMode, env);
+            if (!configuration.compilerPath && !!this.defaultCompilerPath) {
+                configuration.compilerPath = this.defaultCompilerPath;
```

Comment:
`!configuration.compilerPath` will be true when compilerPath is set to "".  We should not be overwriting `""` with `this.defaultCompilerPath`. That will break folks who depend on the `""` behavior.